includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -2846,6 +2874,7 @@ public static function handle_validation_error_update( $redirect_to, $action, $t
 
 		if ( $updated_count ) {
 			delete_transient( AMP_Validated_URL_Post_Type::NEW_VALIDATION_ERROR_URLS_COUNT_TRANSIENT );
+			delete_transient( self::TRANSIENT_KEY_ERROR_INDEX_COUNTS );
```

Comment:
This is not needed with adding a `delete_amp_validation_error` action callback above.